bin/p3.go

```diff
@@ -0,0 +1,55 @@
+package main
```

Comment:
This executable is intended just for convenience when manually testing pathio. Definitely not a replacement for s3cmd. I'll look for examples and move it into a cmd directory.